src/recommendations/product-recs.md

```diff
@@ -0,0 +1,53 @@
+---
+group: product-recommendations
+title: Product Recommendations
+---
+
```

Comment:
This appears to be the front page for Product Recs. I would expect more big picture info, including a mention of Adobe Sensei. Perhaps explain why someone would want to be part of the EAP.